nodejs/tfwconnector/Utilities.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+let MessageSender = require('./MessageSender');
+
+
+class Utilities{
+
+    constructor(messageSender){
```

Comment:
This should use a `TFWServerConnector` instance, not a `MessageSender`, and should initialise one if the user does not provide one as a parameter.